### ISSUE TITLE
[Bug]: Title bar suddently missing on all videos

### ISSUE THREAD

**Issue by Guigui220D** [FreeTubeApp/FreeTube#4104]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

Open any video
No title bar, buttons, channel, etc...
Same symptoms as #1911 but on every video.

![image](https://github.com/FreeTubeApp/FreeTube/assets/24529384/6750f725-539b-4934-90f3-2740f432621b)

### Expected Behavior

The title bar should display

### Issue Labels

visual bug

### FreeTube Version

v0.19.0 Beta

### Operating System Version

Windows 11 Professionnel, 22H2

### Installation Method

.exe

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

it worked on that same version just yesterday

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by Guigui220D** [comment]
Edit: I started the nightly build and it worked in the nightly build portable... and now the main app is back to normal (although it didn't work after several restarts before)

**Comment by Cyanide-1** [comment]
yeah im having this issue too, happened after i updated my arch linux system last night

**Comment by averagedebianuser** [comment]
Same issue here, Arch, installed via AUR, not sure when it happened

**Comment by Cyanide-1** [comment]
> Same issue here, Arch, installed via AUR, not sure when it happened

doesnt seem to be arch-specific either since it's happening on windows from the sounds of it

**Comment by absidue** [comment]
YouTube made a change to their published date format, which broke FreeTube's parsing of it.

Generally if something suddenly breaks in FreeTube, it's because YouTube has changed stuff.

There is already a pull request open with a fix (has been open since after the first comment on this issue, it's linked so you it's visible in the feed), so thanks for your extra comments but the issue has already been identified and fix created.
